.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: "🐞 Bug report"
+about: Create a report to help us improve
+labels: kind/bug
+---
+
+> Bugs should be filed for issues encountered whilst operating the Tekton Dashboard.
+> You should first attempt to resolve your issues through the community support channels,
+> e.g. `#dashboard` channel in the [Community Slack](https://github.com/tektoncd/community/blob/master/contact.md#slack).
+> Please provide as much detail as possible.
+
+# Describe the bug
+
+A clear and concise description of what the bug is.
```

Comment:
Similar for the 'A clear and concise description...' placeholder here and below? (use comment)

Not sure what other folks think, but I often see issues with the placeholder content still in place and it makes it harder to quickly scan the issue to see what's wrong / which sections can be ignored (e.g. additional content)

Reply:
Same thing on the feature request template i suppose ?

Reply:
yes please 👍

Reply:
fixed